user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Tables/Table_component/Filtering_Table_Content.md

```diff
@@ -0,0 +1,85 @@
+---
+uid: Filtering_Table_Content
+---
+
+# Filtering table content
+
+From DataMiner 10.2.7/10.3.0 onwards, you can filter the contents of a table component using one of the available three methods: the [general filter](#general-filter), the [column-based filter](#column-based-filter), or a [text string](#filter-based-on-text-string).
+
+## General filter
+
+To apply a general filter across the table, a **search box** is available:
+
+1. Click the search icon in the top-right corner.
+
+1. Specify the filter text (case-insensitive) in the search box.
+
+   ![General filter](~/user-guide/images/General_Filter.gif)<br>*Table component in DataMiner 10.5.6*
+
+   This will apply a client-side filter only. To apply a server-side filter, you need to use a filter operator when you [configure the query data source](xref:Creating_GQI_query).
```

Comment:
Also important to note is that this is filtering on display values only. Server-side filtering filters on the effective values stored in the database. So searching for gigabyte (GB) works with client side searching, while server-side searching expects > 1 000 000 kb.